x/deployment/types/msgs.go

```diff
@@ -19,12 +21,13 @@ var (
 )
 
 // NewMsgCreateDeployment creates a new MsgCreateDeployment instance
-func NewMsgCreateDeployment(id DeploymentID, groups []GroupSpec, version []byte, deposit sdk.Coin) *MsgCreateDeployment {
+func NewMsgCreateDeployment(id DeploymentID, groups []GroupSpec, version []byte, deposit sdk.Coin, depositor string) *MsgCreateDeployment {
```

Comment:
When passing in an argument that is an account address, we should always try and use one of the cosmos types that actually represents an account address. Strings are problematic when used

Reply:
Done